internal/protoplugin/protoplugin.go

```diff
@@ -159,6 +159,16 @@ type Service struct {
 	Methods []*Method
 }
 
+// FQSN returns a fully qualified service name of this service.
```

Comment:
can we add that to the comment here?  Just for clarity in the code when someone else comes along.